src/metadata.c

```diff
@@ -15,11 +15,11 @@
 #include <utils/datum.h>
 
 #include "catalog.h"
-#include "telemetry_metadata.h"
+#include "metadata.h"
 #include "scanner.h"
 
 #define TYPE_ERROR(inout, typeid)                                                                  \
-	elog(ERROR, "ts::telemetry_metadata: no %s function for type %u", inout, typeid);
+	elog(ERROR, "ts::metadata: no %s function for type %u", inout, typeid);
```

Comment:
Maybe change this to `ts_metadata` now that we have decided on that prefix.

Reply:
Done